nservicebus/operations/metrics.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Metrics
+summary: Measuring the performance and health of an endpoint.
+reviewed: 2017-03-24
+component: Metrics
+related:
+ - samples/metrics
+---
+
+Metrics collects information to measure endpoint health and performance.
+
+
+## Enabling NServiceBus.Metrics
+
+snippet: Metrics-Enable
+
+
+## Reporting metrics data
+
+Metrics can be reported to a number of different locations. Each location is updated on a separate interval. If an interval is not specified then the default interval of 30 seconds is used. To change the default interval:
+
+snippet: Metrics-DefaultInterval
+
+### To NServiceBus log
+
+Metrics data can be written to the [NServiceBus Log](/nservicebus/logging/).
+
+snippet: Metrics-Log
+
+NOTE: Metrics will be written to the log at the INFO log level.
```

Comment:
@SzymonPobiega @mikeminutillo - because we write a lot of info, every 30 seconds, should we set the log level to DEBUG instead of INFO?